tests/unit/src/background/utils/overwriteWitnetNodeConfiguration.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('overwriteConfigFile', () => {
     )
     expect(fs.writeFileSync).toBeCalledWith(
       'sheikah_path/witnet_config_file_name',
-      'node_url = ["20.126.70.77:21338","public_node_url2"]',
+      'node_url = ["public_node_url1","public_node_url2"]\n',
```

Comment:
This test is called `it('Should overwrite the outdated node urls')`, so I believe it was to ensure that only public nodes have their address changed, because users are allowed to set other nodes in this field, and in that case we don't want to delete their node addresses.

Reply:
Sheikah overwrites all node addresses every time there is a new Sheikah update, just in case the version is not compatible. Anyway, this test description is not accurate. I will change it to `Should overwrite a configuration with several nodes according to the parameters given`